WowPacketParser/Enums/ObjectType.cs

```diff
@@ -2,17 +2,20 @@ namespace WowPacketParser.Enums
 {
     public enum ObjectType
     {
-        Object        = 0,
-        Item          = 1,
-        Container     = 2,
-        Unit          = 3,
-        Player        = 4,
-        GameObject    = 5,
-        DynamicObject = 6,
-        Corpse        = 7,
-        AreaTrigger   = 8,
-        SceneObject   = 9,
-        Conversation  = 10,
-        Map           = 11
+        Object                  = 0,
+        Item                    = 1,
+        Container               = 2,
+        AzeriteEmpoweredItem    = 3,
+        AzeriteItem             = 4,
+        Unit                    = 5,
+        Player                  = 6,
+        ActivePlayer            = 7,
+        GameObject              = 8,
+        DynamicObject           = 9,
+        Corpse                  = 10,
+        AreaTrigger             = 11,
+        SceneObject             = 12,
+        Conversation            = 13,
+        Map                     = 14
```

Comment:
Use a different enum you're going to break all the old code

Reply:
fixed